### PR TITLE
Setup fixups

### PR DESCRIPTION
**Pull request by stevejalim**
This changeset
* Updates the Dockerfile's build deps for builder and prod images to allow build and tests to run
  Added:
  * libffi-dev - for cffi
  * zlib1g-dev - for Pillow
  * libjpeg-dev - for Pillow 

* Updates README.md to include env setup step and remove reference to (non-existent) create_locales() helper

### CODE REVIEW

**Comment by stevejalim** [comment]
@pmac - I'm splitting this work into smaller PRs to make it easier to review. I'm still surprised I needed to add some build deps for the buster image, so def worth some extra eyes

**Comment by stevejalim** [comment]
@pmac  OK this is updated again to target Apple Silicon and (I think, based on searching around) this will fall back to non-ARM arch for intel/AMD.  

Could you run `make clean pull build test` on your Intel mac to confirm, please?

**Review by pmac — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -60,12 +62,11 @@ $ docker-compose build
...
-
 ## Install Therapist
 
 [Therapist](https://github.com/rehandalal/therapist) is a smart pre-commit hook
 for git to ensure that committed code has been properly linted.
 
 Install the hooks by running:
 
- `$ therapist install`
+`$ therapist install`
```
[the remark this answers is not in the document]
Comment (stevejalim) [reply README.md:72]: Yeah, agreed - depends on what the road ahead looks like for the service

[the remark this answers is not in the document]
Comment (stevejalim) [reply README.md:72]: It's only running flake8, so not a heavy pre-commit situation

**Pull request merged**